<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Первое число:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="0" column="4" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Второе число:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7" colspan="2">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Операции</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6" colspan="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>//</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Результат:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
